apps/crn-server/test/fixtures/events.fixtures.ts

```diff
@@ -47,6 +47,28 @@ export const getContentfulUserSpeakerTeams = () => ({
   role: 'Lead PI (Core Leadership)',
 });
 
+export const getContentfulRelatedResearch = () => ({
```

Comment:
could you not have used this instead https://github.com/yldio/asap-hub/blob/e4e5b499ff81c04a40b3db3cce5d68b3b60bf643/apps/crn-server/test/fixtures/research-output.fixtures.ts#L505

i think the mock server doesn't mind returning extra properties